Documentation/index.md

```diff
@@ -0,0 +1,17 @@
+# IBM® block storage CSI driver 1.5.0 Release Notes® content map
+
+-   [What's new in 1.5.0](csi_rn_whatsnew.md)
```

Comment:
how is this different from `SUMMARY.md`? is it meant to be always for the previous version?

Reply:
i wouldn't review the index. the content map is supposed to be for the (PDF) book files that I'm still working on seeing how/ if we can still create.

Reply:
ok, then consider removing all not-for-review content from this PR

Reply:
i have to see if this can be removed or if i need it for publishing. i'll play with it and see.